spec/shopify_transporter/exporters/magento/soap_spec.rb

```diff
@@ -43,14 +43,14 @@ def stub_login_call(mock_client)
         end
 
         describe '#call' do
-          it 'initializes savon with the right parameters' do
+          it 'raises FailedLoginError if login response does not contain a session id' do
```

Comment:
do we no longer find value in a test that checks `things initialized properly with the right params`?

Reply:
I think the test case where`things initialized properly with the right params` is included in the next case `creates a session correctly if login response contains a session id`. Correct me if I'm wrong.

Reply:
Since we now need the login response body to have a certain structure in order for things to initialize properly: if we were to have a test that tested that, it would have to stub the login response, which means it would look exactly like the test that starts on line 50.

Reply:
Yep, exactly what Ziyan said ^.